src/e2e-test/java/io/cdap/plugin/odp/stepsdesign/DesignTimeODPError.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.odp.stepsdesign;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.plugin.odp.locators.ODPLocators;
+import io.cdap.plugin.odp.utils.CDAPUtils;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import stepsdesign.BeforeActions;
+
+/**
+ * DesignTimeODPError.
+ */
+public class DesignTimeODPError {
+  static boolean errorExist = false;
+  static String color;
+
+  @Then("{string} as {string} and getting {string}")
+  public void userIsAbleToSetParameterAsAndGetting(String arg0, String input, String errorMessage) {
+    WebElement elementIn = SeleniumDriver.getDriver().findElement(
+      By.xpath("//*[@data-cy='" + arg0 + "' and @class='MuiInputBase-input']"));
+    CDAPUtils.clearField(elementIn);
+    ODPLocators.validateButton.click();
+    color = ODPLocators.rowError.getCssValue("border-color");
+    errorExist = CDAPUtils.getErrorProp(errorMessage).toLowerCase().contains(ODPLocators.rowError.getText()
+                                                                               .toLowerCase());
+  }
+
+  @Then("User is able to validate the validate the error")
+  public void userIsAbleToValidateTheValidateTheError() {
+    Assert.assertTrue(errorExist);
```

Comment:
this assertion can be moved into userIsAbleToSetParameterAsAndGetting().

Reply:
done